frontend/src/tests/data.js

```diff
@@ -27,13 +27,16 @@ export const budgetLine = {
     proc_shop_fee_percentage: 0,
     services_component_id: 1,
     status: "DRAFT",
+    on_hold: false,
+    certified: false,
+    closed: false,
     team_members: [
         '{email: "chris.fortunato@example.com", full_name: "…}',
         '{email: "Amelia.Popham@example.com", full_name: "Am…}',
         '{email: "admin.demo@email.com", full_name: "Admin D…}',
         '{email: "dave.director@email.com", full_name: "Dave…}'
     ],
-    updated_on: "2024-05-27T13:56:50.363964Z"
+    updated_on: "2024-05-27T13:56:50.363964Z",
 };
 
 export const agreement = {
```

Comment:
- ⭐ thanks for keeping our test data up-to-date
- it would be cool to create a type definition file soon for agreements and budget lines

Reply:
Agreed, a type definition would be good for us to make and update. I think I'll be pulling a bunch of the BudgetLineItem tasks off the stack so maybe I will make the type in one of those change lists.